src/test/java/GradleHandlerTest.java

```diff
@@ -19,9 +20,13 @@ public void testGradleProjectBuildSuccess() {
             String projectPath = "GradleTestProjects/CompilableProject";
             // https://stackoverflow.com/questions/28673651/how-to-get-the-path-of-src-test-resources-directory-in-junit
             File projectDirectory = new File(getClass().getClassLoader().getResource(projectPath).getFile());
-
+            Mailserver mailserver = new Mailserver();
+            SendMail sendmail = new SendMail();
             // Act
             Report report = GradleHandler.build(projectDirectory);
+            //CHANGE THE TO- ADDRESS WHEN PAYLOAD HAVE CORRECT STRUCTURE;
+            MimeMessage message = sendmail.SendMail(report, mailserver, "dd2480.lab2.group22@gmail.com,","HELLO MAILBOX!");
+
```

Comment:
I think this class should only test `GradleHandler`. We could move this to a class which tests the clone, build and mail processes at once.

Reply:
Removed all but one. Left the test in case we wanna check how the message looks like until we integrate function into the main process.